00_Base/src/index.ts

```diff
@@ -134,11 +134,9 @@ import {
 } from './ocpp/model/index';
 import { CallAction } from './ocpp/rpc/message';
 
-export interface OcppRequest {
-}
+export interface OcppRequest {}
```

Comment:
Just a question, when I run lint-fix, it will format the code in the way of red line 137 and 138. Maybe you have an idea of which setting cause this?

Reply:
😮 That is very strange, are you on a windows machine?

Reply:
😢 No, I use Mac M1.

Reply:
Just ran with linter and got no changes.

Reply:
Should be sth wired only on my machine. Never mind.